bsd/models.py

```diff
@@ -5,30 +5,78 @@
 from django.core.exceptions import ValidationError
 from django.db import models
 from localflavor.us.models import USStateField
+from StringIO import StringIO
+from xml.etree.ElementTree import ElementTree
 from .api import BSD
 import datetime
 import logging
 import json
 
-
 logger = logging.getLogger(__name__)
 
 """Get BSD api"""
-bsdApi = BSD().api
+bsd_api = BSD().api
 
 BSD_BASE_URL = settings.BSD_BASE_URL
 
 duration_type_minutes = 1
 duration_type_hours = 2
 
 
+def assert_valid_account(api_result):
+    """
+    Assert that api result from BSD has 200 response and contains a constituent
+    with valid account
+    """
+    assert api_result.http_status is 200
+    tree = ElementTree().parse(StringIO(api_result.body))
+    cons = tree.find('cons')
+    assert cons is not None
+    cons_id = cons.get('id')
+    assert cons_id is not None
+    assert cons.find('has_account').text == "1"
+    assert cons.find('is_banned').text == "0"
+
+
 def get_bsd_event_url(event_id_obfuscated):
     return "%s/page/event/detail/%s" % (
         BSD_BASE_URL,
         event_id_obfuscated
     )
 
 
+class Account(models.Model):
+    """BSD User Account"""
+    email_max_length = 128
+    name_max_length = 128
+    password_max_length = 100
+    postal_code_max_length = 16
+    email_address = models.EmailField(max_length=email_max_length)
+    first_name = models.CharField(max_length=name_max_length)
+    last_name = models.CharField(max_length=name_max_length)
+    password = models.CharField(max_length=password_max_length)
```

Comment:
I assume this will be storing a hashed password? Should we specify here?

Reply:
I'm not sure what you mean, we don't store it, we are just sending it to bsd api?

Reply:
Aren't these models stored in DB?

Reply:
No. `managed = False` means do not store in db.

Reply:
Ahhhh gotcha, missed that, thanks!